docs/source/API/core/utilities/complex.rst

```diff
@@ -1,177 +1,295 @@
-``Kokkos::complex``
-===================
+``complex``
+===========
 
 .. role:: cpp(code)
     :language: cpp
 
-Header File: ``<Kokkos_Core.hpp>``
+Defined in header ``<Kokkos_Complex.hpp>`` which is included from ``<Kokkos_Core.hpp>``
 
-Usage
------
+Description
+-----------
 
-.. code-block:: cpp
+``complex`` is a class template for representing and manipulating complex numbers.
 
-    Kokkos::complex<double> a,b;
-    a.imag() = 5.0; a.real() = 1.0
-    b = a;
-    a += b;
+* This is intended as a replacement for ``std::complex<T>``.
+* Note: If ``z`` has type ``Kokkos::complex<T>``, casting such as ``reinterpret_cast<T(&)[2]>(z)`` leads to undefined behavior (this differs from ``std::complex``).
 
+Interface
+---------
 
-Description
------------
+.. cpp:class:: template<class T> complex
+
+  .. rubric:: Template Parameters
+
+  :tparam T: The type of the real and imaginary components.
```

Comment:
<img width="806" alt="Screenshot 2025-06-06 at 9 14 51 PM" src="https://github.com/user-attachments/assets/95c4d78c-27bf-467c-b21f-63ab14d7e73d" />
It yields duplicated "Template Parameters"